devel/hg-git/Portfile

```diff
@@ -2,10 +2,9 @@
 
 PortSystem              1.0
 PortGroup               python 1.0
-PortGroup               gitlab 1.0
 
-gitlab.instance         https://foss.heptapod.net
```

Comment:
Are you sure that we should revert to pypi? The reason that we were building from source from the canonical location and not always using pypi is because this project had a history of having very infrequent releases, and the extension would break on newer versions of Mercurial, and fixes would be committed in `master` but not released for a long time.

Reply:
I was made co-maintainer of hg-git not long ago, and I actually pushed the most recent release to PyPI myself. So if it's out of date sometime in the future, please do complain 🙂

Reply:
All right then. :-) Good to hear.